src/Data/Singletons/Prelude/Monad/Internal.hs

```diff
@@ -44,16 +45,6 @@ import Data.Singletons.Prelude.Base
 import Data.Singletons.Prelude.Instances
 import Data.Singletons.Single
 
-{-
```

Comment:
Actually, this Note has been obsolete ever since #416 (which did indeed delete various kind annotations), but I somehow skipped over this Note at the time. I decided to posthumously remove it in an act of "while-I'm-in-town" cleanup.